atsign-foundation/at_java
# retrying otp does not call onboarding

closes https://github.com/atsign-foundation/at_java/issues/14
**- What I did**

- Solved a bug where re-entering otp does not call onboarding
- Handled a case when user exhausts their quota of free atsigns by displaying appropriate message
- Also restructure config yaml and changed the config variables to camelCase

**- How I did it**

- reposition if-else blocks so that the block calling onboarding class is executed every time
- calling API to check if they have exhausted free atsign quota before trying to register it to a users email

**- How to verify it**

- use register CLI and enter incorrect otp and then enter correct otp. Expected behaviour is that onboarding is successfully completed
- try registering a new atsign with an email that has exhausted free atsign limit, the CLI should throw an exception "free atsign limit exhausted"

**- Description for the changelog**
